scripts/build-glean-metadata.py

```diff
@@ -227,6 +241,9 @@ def get_app_variant_description(app):
     if all([app_id.get("prototype") for app_id in app_group["app_ids"]]):
         app_data["prototype"] = True
 
+    app_annotation = _get_annotation(annotations_index, app_name, "app")
+    app_tags = app_annotation.get("tags")
```

Comment:
Do we need to provide a fall back value for `app_annotation.get("tags")` or does `tags` always exist?

Reply:
If the app has no tags then there shouldn't be any annotations with them: it may be best to fallback to `{}` in this case just for clarity though. I'll change this.